droidlet/lowlevel/locobot/remote/slam_pkg/slam.py

```diff
@@ -304,6 +359,34 @@ def take_step(self, step_size):
             return False
         return None
 
+    def save_rgb_depth_seg(self):
+        rgb, depth, seg = self.robot.camera.get_rgb_depth_segm()
+        pos = self.robot.base.get_state()
+        self.skp += 1
+        if pos != self.last_pos and self.skp % 10 == 0:
+            self.last_pos = pos
+            # store the images and depth
+            cv2.imwrite(
+                self.img_folder + "/{:05d}.jpg".format(self.img_count),
```

Comment:
i learnt over the years that it's a lot more robust to use `os.path.join`, because if one accidentally forgets slashes somewhere, it still does the right thing.
`os.path.join(self.img_folder, "{:05d}.jpg".format(self.img_count)`